packages/cardano-services/README.md

```diff
@@ -1,7 +1,92 @@
 # Cardano JS SDK | Cardano GraphQL Services
+Libraries and program entrypoints for services that facilitate remote data and submit access using 
+[Provider] interfaces over HTTP, with _optional_ queue-based transaction submission; The 
+[TxSubmitHttpService] can be configured to submit directly via [Ogmios], or via a RabbitMQ broker,
+with one or more workers handling submission and response job creation.
+
+## Features
+- [CLI] or run scripts for the [HTTP server](./src/run.ts) and [worker](./src/startWorker.ts) with 
+configuration via environment variables and _optional_ loading of secrets from disk.
+- Service port discovery via DNS resolution, or static configuration.
+- Fault-tolerant transaction submission via persistent queue, or direct submission.
+- _Optional_ [Prometheus] metrics available at `/metrics`
+
+## Services
+### HTTP Server
+The HTTP server can be started with one or more provider modules by name, segmented by URL path. 
+Run the [CLI] with `start-server --help` to see the full list of options, or inspect the 
+environment variables within [./src/run.ts](./src/run.ts). 
+
+### Worker
+A worker must be started when opting for queue-based transaction submission.
+Run the [CLI] with `start-worker --help` to see the full list of options, or inspect the
+environment variables within [./src/startWorker.ts](./src/startWorker.ts).
+
+
+## Examples
+
+_The following examples require the [install and build] steps 
+to be completed._
+
+#### All Providers | Static Service Config | Direct Tx Submission
+
+- The server will expose all [Provider] HTTP services
+- Transactions will be submitted directly via [Ogmios], running at `ws://localhost:1338`.
+- Connects to [PostgreSQL] service running at `localhost:5432`
+- HTTP API exposed using a custom API URL
+
+``` console
+./dist/cli.js \
```

Comment:
These end up in `./dist/cjs` folder. By the way, in my build these files don't have execute permissions set.

Reply:
Fixed the invalid path in 3f2085911a5932c3484183befdb0da7d9354e98f, and executable permissions in 1c1eeb2240c98bf53ce5007406e094f82f2a255d